src/Education.UnitTests/Education.UnitTests.csproj

```diff
@@ -0,0 +1,19 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net4.8</TargetFramework>
+
+    <IsPackable>false</IsPackable>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.6.1" />
```

Comment:
All of our test projects except `SonarQube.Client.Tests` references version `16.5.0` of this package (which is unfortunate, since I think that is the test project you looked at). It's better to reference the same version.